pkg/listening-sockets/listening_sockets_test.go

```diff
@@ -76,8 +76,8 @@ var (
 			Port:      111,
 			NodeRole:  "master",
 			Service:   "rpcbind",
-			Namespace: "",
-			Pod:       "",
+			Namespace: "test-namespace",
```

Comment:
What's the reason for this change?

Reply:
now we are returning the namespace and pod name and this will effect the test 
on the test we will have it